apps/cli/src/index.ts

```diff
@@ -17,4 +17,12 @@ program
   .option("--dry-run", "Simulate init")
   .action(actions.init);
 
+program
+  .command("deploy")
```

Comment:
I was thinking, perhaps publish is a better name? What do you think? Deploy might be confused with a command to deploy a link-to-code instance to a new firebase